src/clj/rems/api/licenses.clj

```diff
@@ -39,4 +61,34 @@
       :roles #{:owner}
       :body [command CreateLicenseCommand]
       :return CreateLicenseResponse
-      (ok (licenses/create-license! command)))))
+      (ok (licenses/create-license! command)))
+
+    (POST "/add_attachment" []
+      :summary "Add an attachment file that will be used in a license"
+      :roles #{:owner}
+      :multipart-params [file :- upload/TempFileUpload]
+      :middleware [upload/wrap-multipart-params]
+      :return AttachmentMetadata
+      (check-attachment-content-type (:content-type file))
+      (ok (licenses/create-license-attachment! file (util/getx-user-id))))
+
+    (POST "/remove_attachment" []
+      :summary "Remove an attachment file related to an application field"
+      :roles #{:owner}
+      :query-params [attachment-id :- (describe s/Int "attachment id")]
+      :return SuccessResponse
+      (if (some? (licenses/remove-license-attachment! attachment-id))
+        (ok {:success true})
+        (ok {:success false})))
+
+    (GET "/attachments/:attachment-id" []
```

Comment:
why attachment-id and not license-id?

oh right, because you first create the attachment and then the license.